website/docs/reference/node-selection/syntax.md

```diff
@@ -103,9 +103,21 @@ As your selection logic gets more complex, and becomes unwieldly to type out as
 consider using a [yaml selector](/reference/node-selection/yaml-selectors). You can use a predefined definition with the `--selector` flag.
 Note that when you're using `--selector`, most other flags (namely `--select` and `--exclude`) will be ignored.
 
+### Troubleshoot with the `ls` command
+
+Constructing and debugging your selection syntax can be challenging.  To get a "preview" of what will be selected, we recommend using the [`list` command](/reference/commands/list).  This command, when combined with your selection syntax, will output a list of the nodes that meet that selection criteria.  The `dbt ls` command supports all types of selection syntax arguments, for example:
+
+```bash
+dbt ls --select "path.to.my.models" # Lists all models in a specific directory.
```

Comment:
should it be this?
```suggestion
dbt ls --select "path/to/my/models" # Lists all models in a specific directory.
```

Reply:
I copied that from another example on the page. But I agree that slashes makes more sense

Reply:
awesome, thank yoU! i'll try to find the other one and update it too! 🙏